app/views/CreditAndRefunds.scala.html

```diff
@@ -41,29 +42,29 @@
     govukInsetText : GovukInsetText
 )
 @(viewModel: CreditAndRefundViewModel, balance: Option[BalanceDetails], creditAndRefundType: Option[UnallocatedCreditType],
-    isAgent: Boolean = false, backUrl: String, isMFACreditsAndDebitsEnabled: Boolean = false, isCutOverCreditsEnabled: Boolean = false)(implicit request: Request[_], user: MtdItUser[_], messages: Messages)
+isAgent: Boolean = false, backUrl: String, isMFACreditsAndDebitsEnabled: Boolean = false, isCutOverCreditsEnabled: Boolean = false)(implicit request: Request[_], user: MtdItUser[_], messages: Messages)
```

Comment:
I think we would need a story to refactor next under viewModel definition
`isAgent: Boolean = false, backUrl: String, isMFACreditsAndDebitsEnabled: Boolean = false, isCutOverCreditsEnabled: Boolean = false`

Reply:
Could you expand please

Reply:
I am not sure about agent and backUrl parameters, but 
isMFACreditsAndDebitsEnabled and isCutOverCreditsEnabled can be moved into viewModel, 
so list of parameters we are passing to the view will become "lighter", but I think Ian was thinking to split such views into view, on per each FS (?)